capstone-frontend/src/components/ListPage.jsx

```diff
@@ -30,7 +30,7 @@ class ListPage extends Component {
     const gBooks = [];
 
     await Promise.all(gbookIDs.map(async (gBookID) => {
-      const gBook = await fetch(`https://www.googleapis.com/books/v1/volumes/${gBookID}`).then(response => response.json())
+      const gBook = await fetch(`/api/search?gbookId=${gBookID}`).then(response => response.json())
       gBooks.push(gBook);
```

Comment:
You could push `gBook[0]` here rather than having to reference `gBook[0]` everywhere else.

Reply:
Done.